spring-webflux/src/main/java/org/springframework/web/reactive/function/client/ClientRequest.java

```diff
@@ -125,8 +125,8 @@
 	// Static builder methods
 
 	/**
-	 * Create a builder with the method, URI, headers, and cookies of the given request.
-	 * @param other the request to copy the method, URI, headers, and cookies from
+	 * Create a builder with the method, URI, headers, cookies and body of the given request.
+	 * @param other the request to copy the method, URI, headers, cookies and body from
```

Comment:
```suggestion
	 * Create a builder with the method, URI, headers, cookies, attributes, and body of the given request.
	 * @param other the request to copy the method, URI, headers, cookies, attributes, and body from
```

Reply:
Thank you for your review and suggested changes. I will address them this evening.